AudibleApi.Common/AudibleApi.Common.csproj

```diff
@@ -12,4 +12,12 @@
     <PackageReference Include="Dinah.Core" Version="4.4.1.1" />
   </ItemGroup>
 
+	<PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
Does this mean that Libation logs will show line numbers from the audibleapi nuget dlls? That would be outstanding

Reply:
Yes it does!